cypress/integration/integration_spec.js

```diff
@@ -70,17 +70,6 @@ it('should 404 unknown routes', () => {
     check404('/not/a/page');
 });
 
-it('should redirect search queries to a google site search', () => {
```

Comment:
This test has been moved to a unit test with jest. Made more sense as we don't invoke a browser UI here.